tests/unit/test_cmdoptions.py

```diff
@@ -51,3 +57,110 @@ def test_identify_python_interpreter_venv(tmpdir: Path) -> None:
 
     # Passing a non-existent file returns None
     assert identify_python_interpreter(str(tmpdir / "nonexistent")) is None
+
+
+@pytest.mark.parametrize(
+    "value, expected_check",
+    [
+        # Test with timezone info (should be preserved exactly)
+        (
+            "2023-01-01T00:00:00+00:00",
+            lambda dt: dt
+            == datetime.datetime(2023, 1, 1, 0, 0, 0, tzinfo=datetime.timezone.utc),
+        ),
+        (
+            "2023-01-01T12:00:00-05:00",
+            lambda dt: (
+                dt
+                == datetime.datetime(
+                    *(2023, 1, 1, 12, 0, 0),
+                    tzinfo=datetime.timezone(datetime.timedelta(hours=-5)),
+                )
+            ),
+        ),
+    ],
+)
```

Comment:
Ugh, I am not a fan of how Black explodes large collection literals. Could we instead only pass the expected datetime object via the decorator and perform the equality check in the test?

Reply:
Fixed, as best as I could